wallet/src/Pos/Wallet/Web/Methods/Payment.hs

```diff
@@ -69,14 +72,30 @@ newPayment
     -> Coin
     -> InputSelectionPolicy
     -> m CTx
-newPayment sa passphrase srcAccount dstAccount coin policy = do
+newPayment sa passphrase srcAccount dstAccount coin policy =
     sendMoney
         sa
         passphrase
         (AccountMoneySource srcAccount)
         (one (dstAccount, coin))
         policy
 
+newPaymentBatch
+    :: MonadWalletWebMode m
+    => SendActions m
+    -> PassPhrase
+    -> NewBatchPayment
+    -> m CTx
+newPaymentBatch sa passphrase NewBatchPayment {..} = do
+    src <- decodeCTypeOrFail npbFrom
+    notFasterThan (6 :: Second) $
```

Comment:
Hmm, check that we've not merged this already. I thought that we already had `notFasterThan` in master and were using it for `newPayment`.

Reply:
@dcoutts As surprising as it seems, we don't have (it might be introduced by some of the ported outstanding PRs, but not the the current tip):

![screen shot 2018-01-10 at 15 30 56](https://user-images.githubusercontent.com/29383371/34777676-5d273d98-f61b-11e7-95e9-3d75a9bbddc6.png)

Reply:
Ok, find it in the exchanges branch. We need to include that change.

Reply:
Do we really need this? We already have rate limiting in the dequeue policy. @dcoutts